gzip/README.md

```diff
@@ -0,0 +1,21 @@
+# gzip 
+
+Build the [gzip](https://www.gnu.org/software/gzip/) data compressor as a
+static PIE ELF running on Linux.
+
+## Building
+
+The `gzip` static PIE ELF file is located in the current directory.
+If you want to rebuild it, run:
+
+```
+$ ./build.sh
+```
+
+This script downloads, unpacks, patches, configures and builds the `gzip`
```

Comment:
Please use one sentence per line and one line per sentence, since it's easier to review and the github markdown renderer will space it properly.

Reply:
Removed all the line splitting